exporter/internal/err.go

```diff
@@ -1,7 +1,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package internal // import "go.opentelemetry.io/collector/exporter/exporterhelper/internal"
+package internal // import "go.opentelemetry.io/collector/exporter/internal"
```

Comment:
put it in a `internal/queue` package please.

Reply:
done